mibidata/mibi_image.py

```diff
@@ -91,57 +115,66 @@ class MibiImage():
         instrument: A string identifier for the instrument used.
         tissue: A string name of the tissue type.
         panel: A string name of the panel used to stain the tissue.
-        version: A string identifier for the software version used.
+        version: A string identifier for the MIBItiff software version used.
```

Comment:
Let's confirm that we are not using this for writing instrument version. If we are, or perhaps even if we aren't, we may want to specific`'MIBItiff_version'` so we can also specific instrument version etc.

Reply:
Ok, so I'm adding a new attribute `MIBItiff_version` alongside `version`.

Reply:
@rfinck @jaytarolli @chris-brent Can we rename the old `version` to something like `instrumet_version`? Or have it inside a dictionary `instrument`, with keys `name` and `version`? Otherwise it can be confusing/misleading for the user to see 2 attributes called version.